app/src/main/java/com/e/salesapp/CategoriesListFragment.java

```diff
@@ -12,25 +12,24 @@
 
 public class CategoriesListFragment extends Fragment {
 
-    OnCategorySelected listener;
+    OnCategorySelected mCategoryListener;
```

Comment:
Why you are not defining the access modifier(public or private)?, please avoid use default modifier access